src/components/Workspace.js

```diff
@@ -294,7 +294,8 @@ class Workspace extends Component {
 
     this.updateSensorStateCache(sensor.left, sensor.right,
       sensor.leftLightSensorReading, sensor.rightLightSensorReading,
-      sensor.leftLineSensorReading, sensor.rightLineSensorReading);
+      sensor.leftLineSensorReading, sensor.rightLineSensorReading,
+      sensor.distanceSensorReading);
```

Comment:
It may make sense at this point to just pass in `sensor` and have the function use destructuring on the object

Reply:
Done. We should also reface the names of the button state variables, but I'll leave that for a separate PR since it spans up through the sensor reducer.